gnu-apl-follow.el

```diff
@@ -4,18 +4,24 @@
 (require 'gnu-apl-util)
 (require 'gnu-apl-network)
 
+(declare-function gnu-apl--get-interactive-session "gnu-apl-interactive")
+(declare-function gnu-apl--name-at-point "gnu-apt-documentation")
+(declare-function gnu-apl--choose-variable "gnu-apl-editor"
+                  (prompt &optional type default-value))
+(declare-function gnu-apl--get-interactive-session-with-nocheck "gnu-apt-interactive")
+
 (defun gnu-apl--make-trace-buffer-name (varname)
   (format "*gnu-apl trace %s*" varname))
 
-;;;
-;;;  gnu-apl-trace-symbols is a list of traced symbols, each element
-;;;  has the following structure:
-;;;
-;;;    ("symbol_name" <buffer>)
-;;;
+(defvar gnu-apl-trace-variable)
+
+(defvar gnu-apl-trace-symbols nil
+  "List of traced symbols.
+Each element has the structure (\"symbol_name\" <buffer>).")
 
 (defun gnu-apl-trace-mode-kill-buffer ()
-  "If the current buffer is a trace buffer, kill the buffer. Otherwise raise an error."
+  "If the current buffer is a trace buffer, kill the buffer.
+Otherwise raise an error."
```

Comment:
What is the reason for a newline here? It breaks the flow of the text and when only the first line is displayed, not all information is shown.

Reply:
The reason is that checkdoc complains about documentation strings that are longer than 80 characters. If you want to, I can just ignore checkdoc for now.

Reply:
Yes, please do so in this case.

Reply:
If you insist, but I'll just point out that this is the last remaining byte compilation issue.

Reply:
If you can come up with a better wording, I'm all for it.

Just so you see my concern: The issue is that the entire behaviour right now is concisely covered by a single sentence. Having an arbitrary 80 character limit break that is not ideal.

Reply:
Thinking about this some more, I tried setting `fill-collumn` in .dir-locals.el to a value greater than 80, and that seems to have "fixed" the issue. I'll have to check if ELPA accepts this too, but it seems like a first approximation of a satisfactory solution.